tests/test_avecmondoc.py

```diff
@@ -0,0 +1,324 @@
+import json
+from scraper.pattern.scraper_request import ScraperRequest
+from scraper.pattern.center_location import CenterLocation
+from scraper.pattern.center_info import CenterInfo
+import httpx
+from pathlib import Path
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+from datetime import datetime
+
+from scraper.avecmondoc.avecmondoc import (
```

Comment:
How about:

```python
from scraper.avecmondoc import avecmondoc
```

Comme ça, un seul import, et ça sert même comme namespace, e.g. :

```python
avecmondoc.search(...)
```

what do you think?